efs.tf

```diff
@@ -4,6 +4,14 @@ resource "aws_iam_policy" "efs" {
   policy      = data.aws_iam_policy_document.efs.json
   tags        = local.tags
 }
+
+resource "aws_efs_file_system_policy" "policy" {
```

Comment:
Why this resource?

Reply:
cloudposse was also creating this resource,but it relied on count( without data block)
but the argument of this requires a `policy` which needs to be generated via data block.

example: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/efs_file_system_policy

we had to move this out of that module, to get smooth one-shot plan 
https://github.com/cloudposse/terraform-aws-efs/blob/main/main.tf#L153-L159